### DISCUSSION TITLE
2010/07/28/%E3%80%8A%E6%88%91%E7%9A%84%E5%A4%8D%E5%88%B6%E5%8F%AF%E4%BB%A5%E6%88%90%E5%8A%9F%E3%80%8B

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#291] [Announcements]
# 2010/07/28/%E3%80%8A%E6%88%91%E7%9A%84%E5%A4%8D%E5%88%B6%E5%8F%AF%E4%BB%A5%E6%88%90%E5%8A%9F%E3%80%8B

我本来都没怎么听说过唐骏这人，我平时比较关注相关公司的技术动向，至于这些公司的领导人则不太关心。像《我的成功可以复制》这种没技术内容纯粹吹牛的书更不可能去看了。不过最近唐骏很火，网上到处都是关于他的文章，《我的成功可以复制》也跟着家喻户晓了。昨天恰好看到有网友张贴了《我的成功可以复制》中的一部分内容，和技术有些关系。唐骏号称是一开始去微软就做汉化的，他这段表述，非技术人员就不要看了；技术人员可以当笑话看看，调节一下心情：

https://blog.qizhen.xyz/2010/07/28/%E3%80%8A%E6%88%91%E7%9A%84%E5%A4%8D%E5%88%B6%E5%8F%AF%E4%BB%A5%E6%88%90%E5%8A%9F%E3%80%8B

**Comment by ruanqizhen** [comment]
之前博客下的留言：

茂
2010年08月6日 15:05 编辑
哈哈，不知道是唐骏自己不懂字符编码，还是帮他写书的人没思考，那“微”字怎么拆呢，岂不是要用3个字节来存储:-)
